config/README.md

```diff
@@ -1,6 +1,67 @@
-We use [Firebase Remote Config](https://firebase.google.com/docs/remote-config) for dynamic configuration of our app.
+This project is based on [REST APIs using the node client](https://github.com/firebase/quickstart-nodejs/tree/master/config) to manipulate remote config in firebase.
+
+More information about what we are using remote config for can be found in this [private gdrive document](https://docs.google.com/document/d/1UNAAbFE_U8KVSd5SNg_Z0Gm9Kn7TY2UbQbbsDdz6ZGg)
+
+The project is written using our favourite command / programming languages `javascript` and `bash`, though fear not, the project can easily be converted to other languages, here are some other quickstart examples of firebase remote config:
```

Comment:
Iiik :-).  I'm currently trying to avoid using both, by using Golang instead :-).    That said.  The general idea of using a proper programming language (and I will grudgingly admit that .js is a proper programming language) instead of just a scripting language like Bash is a practice we should encourage and evolve.

Reply:
I agree, there is no need to use both, it was more of an convenience, luckily there are other language options out there, in case someone's fingers are itching to convert :D